types/sanctuary/sanctuary-tests.ts

```diff
@@ -1,10 +1,73 @@
 /// <reference types="node"/>
-import * as assert from "assert";
-import { create, env } from "sanctuary";
+import * as assert from 'assert';
+import { create, env } from 'sanctuary';
 
-const checkTypes = process.env["NODE_ENV"] !== "production";
-const S = create({checkTypes, env});
+const checkTypes = process.env['NODE_ENV'] !== 'production';
+const S = create({ checkTypes, env });
```

Comment:
This is now the default behaviour, so we could simply import `S`. :)

Reply:
👍

Reply:
cool!